dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/ContainerHolder.java

```diff
@@ -0,0 +1,47 @@
+package com.dotmarketing.portlets.templates.design.bean;
+
+import com.dotmarketing.portlets.templates.design.util.PreviewTemplateUtil;
+
+import java.util.List;
+
+/**
+ * Created by freddyrodriguez on 11/13/17.
+ */
+public class ContainerHolder {
+
+    private boolean preview;
+    private List<String> containers;
+
+    public List<String> getContainers() {
+        return containers;
+    }
+
+    public void setContainers(List<String> containers) {
+        this.containers = containers;
+    }
+
+    public boolean isPreview() {
+        return preview;
+    }
+
+    public void setPreview(boolean preview) {
+        this.preview = preview;
+    }
+
+    public String draw () throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.Exception', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

Reply:
it looks like logic in a Value object, the concern should be separated

Reply:
Yes, but it's better change it later

Reply:
why?, IMO if you do not change it now it will stay like this forever :)